### PR TITLE
SG-29417 [Python API Documentation] Update Python version requirements

### PR DESCRIPTION
**Pull request by 000paradox000**
Remove python 2.7 from readme and docs sections "Minimum requirements"

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -9,7 +9,7 @@ Minimum Requirements
 - ShotGrid server v5.4.14 or higher for API v3.0.16+.
```
Comment (dsauve-adsk) [inline docs/installation.rst:9]: Did we say that we need to remove these 3 lines? (or will it be part of another initiative?)

**Review by dsauve-adsk — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ The latest version can always be found at http://github.com/shotgunsoftware/pyth
 ## Minimum Requirements
 
 * ShotGrid server v2.4.12+.
```
Comment (julien-lang) [inline README.md:15]: Can you remove this line, please?

File: docs/installation.rst [hunk 3]
```diff
@@ -9,7 +9,7 @@ Minimum Requirements
...
 - ShotGrid server v2.4.12 or higher for API v3.0.8.
 - ShotGrid server v1.8 or higher for API v3.0.7.
```
Comment (julien-lang) [inline docs/installation.rst:11]: Can you remove these old SG version references?

**Review by julien-lang — CHANGES_REQUESTED** [review]

**Review by julien-lang — APPROVED** [review]
LGTM

**Pull request merged**